app/src/main/java/com/example/keniphhomemaintenance/database/DwellingWithMaintenanceItems.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.keniphhomemaintenance.database
+
+import androidx.room.Embedded
+import androidx.room.Relation
+import com.example.keniphhomemaintenance.dwellings.Dwelling
+import com.example.keniphhomemaintenance.maintenance_screen.MaintenanceItem
+
+data class DwellingWithMaintenanceItems (
```

Comment:
Generally you don't include a description of a class's contents within the class's name. 

Looking more at the code, I don't think there's a good reason to create this class - just add the List<MaintenanceItem> to the existing Dwelling class.